taipy/_cli/_scaffold_cli.py

```diff
@@ -45,6 +46,10 @@ def handle_command(cls):
         args = cls._parse_arguments()
         if not args:
             return
-
-        cookiecutter(cls._TEMPLATE_MAP[args.template])
+        try:
+            cookiecutter(cls._TEMPLATE_MAP[args.template])
+        except OutputDirExistsException as err:
+            error_msg = f"{str(err)[7:]}. Please remove the existing directory or provide a new folder name."
+            cls._logger.error(error_msg)
```

Comment:
Can't we simply get rid of the cookie-cutter exception? Something like:

```suggestion
            error_msg = f"Folder '{output_dir}' already exists. Please remove the existing folder or provide a new folder name."
            print(error_msg)
```

Plus I believe we don't need to use the taipy logger here. There is no application running. We just exit the command line with a message.

Reply:
We can not get the `output_dir` since it's handle by cookiecutter and is only returned if the command success.

Reply:
Oh, right, it is not part of the command line arguments. It is in the answer to the application name question!

So I would suggest:

```suggestion
            error_msg = f"Please remove the existing folder or provide a new folder name."
            print(error_msg)
```

Reply:
That may be to vague though, because we do not mention the folder name in the log. It would look weird, something like:
```
Application root folder name [taipy_application]: a
Application main Python file [main.py]: 
Application title [Taipy Application]: 
Page names in multi-page application? []: 
Does the application use scenario management or version management? [No]: 
Does the application use Rest API? [No]: 
[2024-04-25 14:32:17.716][Taipy][ERROR] Please remove the existing directory or provide a new folder name.
```

There is nothing indicate that the "a" folder exists, but only a message said to remove the existing folder.

Reply:
That's true. And what is the output of this piece of code: `{str(err)[7:]}`?

Reply:
In cookiecutter, the message is defined as
```
f'Error: "{dir_to_create}" directory already exists'
```

So the `[7:]` is to remove the "Error: " part of the message

Reply:
OK, so indeed that's the right message.

```suggestion
            error_msg = f"{str(err)[7:]}. Please remove the existing directory or provide a new folder name."
            print(error_msg)
```

Reply:
I use the `logger.error` so that we can indicate this is an error raised by Taipy. Print would not have the same effect and we should keep the `"Error: "` part of the message

Reply:
OK, I got it now, and I agree. Thank you.

However, since this is for a command line and not for runtime, I would still not use a logger but the print function directly. 
That means we don't need to remove the 'Error' part of the cookiedcutter message.

 ```suggestion
            error_msg = f"{str(err)}. Please remove the existing directory or provide a new folder name."
            print(error_msg)
```

What do you think?

Reply:
SOrry about that. We sent our messages at the same time.

Reply:
Yes that makes sense